### ISSUE TITLE
any help opened this issue 13 days ago 

### ISSUE THREAD

**Issue by zayoood** [Globox1997/AdventureZ#71]
Hello, I have a slightly strange problem. The mod works fine, but I can't find any mobs in the map. I only find 2 (Necromancer and Summoner) I can't find anyone in the whole map. I think the problem is because I haven't started in a new map, so it may not work perfectly. ?

**Comment by zayoood** [comment]
any help ?

**Comment by Globox1997** [comment]
Sorry, sometimes I oversee some issues. Does it happen with only the latest version and what biome mods do you have installed?

**Comment by ThatCarrot** [comment]
Hi i'm kinda having the same issue.

I generated a new world with terralith and installed the mod after.
Only ever seen an Orc and a Summoner.
Via execute command i checked if any amethyst golems existed in already generated area, got nothing.
I went to a mushroom island exclusivly to see if any mushroom mobs spawned, nothing. This island got generated with adventure z mod loaded.
Never seen a deer.
Went to the Nether, dug some gold and piglins got mad, but no pigling beast.
Never seen any other nether mob.
The world is 60 in-game days old, so, **i guess**, i should be seeing more mobs.

However, i went to non generated area (so world generates with adventurez loaded) and found significant amout of amethyst golems.

```
Game Version 1.18.1, Fabric, Multiple Mods, including Terralith and Better Nether.
Mod Version 1.4.7
```
